src/LaravelBook/Ardent/Ardent.php

```diff
@@ -538,7 +538,9 @@ public function validate(array $rules = array(), array $customMessages = array()
 
 			if ($success) {
 				// if the model is valid, unset old errors
-				if ($this->validationErrors->count() > 0) {
+				if($this->validationErrors === null){
+					$this->validationErrors = new MessageBag;
+				}elseif($this->validationErrors->count() > 0) {
```

Comment:
Pease, fix spacing. Would be nice if you follow PSR-2: http://www.php-fig.org/psr/psr-2/#5.1.-if,-elseif,-else